From 3f5175e56c5678fd06300d6336795ff548b3a556 Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Thu, 19 Nov 2009 03:33:52 +0100
Subject: [PATCH] Fix build system auto-selection breakage.

Probably due to an overlook in 758ce0bb1f, the '-e' test on build.xml
disappeared, leading check_auto_buildable() to always return '1' for
the ant build system.

Signed-off-by: Cyril Brulebois <kibi@debian.org>
---
 Debian/Debhelper/Buildsystem/ant.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Debian/Debhelper/Buildsystem/ant.pm b/Debian/Debhelper/Buildsystem/ant.pm
index 938fb44..52def4f 100644
--- a/Debian/Debhelper/Buildsystem/ant.pm
+++ b/Debian/Debhelper/Buildsystem/ant.pm
@@ -14,7 +14,7 @@ sub DESCRIPTION {
 
 sub check_auto_buildable {
 	my $this=shift;
-	return $this->get_sourcepath("build.xml") ? 1 : 0;
+	return (-e $this->get_sourcepath("build.xml")) ? 1 : 0;
 }
 
 sub new {
-- 
1.6.5.2

